src/Client.php

```diff
@@ -365,7 +365,7 @@ public function __set($name, $value)
     }
 
     /**
-     * @param $name
+     * @param        $name
```

Comment:
Let's use the type that the php documentation suggests here.

Reply:
Done